### PR TITLE
Martin dev

### PR DESCRIPTION
**Pull request by mharcar**

### CODE REVIEW

File: src/Agent/MyStatistics.vue [hunk 1]
```diff
@@ -15,6 +15,30 @@
...
+            <div style="text-align:center; font-weight: normal; color:#969696">{{data.label}}</div>
+          </template>
+          <template slot="HEAD_statesCounts" scope="data">
+            <div style="text-align:center; font-weight: normal; color:#969696">{{data.label}}</div>
+          </template>
+          <template slot="HEAD_myCpt" scope="data">
+            <div style="text-align:center; font-weight: normal; color:#969696">{{data.label}}</div>
+          </template>
+          <template slot="HEAD_teamCpt" scope="data">
+            <div style="text-align:center; font-weight: normal; color:#969696">{{data.label}}</div>
```
Comment (jamhed) [inline src/Agent/MyStatistics.vue]: Can you put this style to class?

Comment (mharcar) [reply src/Agent/MyStatistics.vue]: sure

File: src/Agent/MyStatistics.vue [hunk 2]
```diff
@@ -15,29 +15,29 @@
         <b-table style="margin-top:10px" small bordered
           :items="statistics"
           :fields="fields">
-          <template slot="HEAD_ciq" scope="data">
-            <div style="text-align:center; font-weight: normal; color:#969696">{{data.label}}</div>
+          <template slot="HEAD_ciq" slot-scope="data">
+            <div class="table-header-text-center">{{data.label}}</div>
```
Comment (jamhed) [inline src/Agent/MyStatistics.vue]: Better to use b-table thClass, don't you think? Also, why do you need slots for headers at all?

Comment (mharcar) [reply src/Agent/MyStatistics.vue]: because it was one possible option

**Pull request merged**